test/models/batch_invitation_test.rb

```diff
@@ -5,7 +5,10 @@ class BatchInvitationTest < ActiveSupport::TestCase
     ActionMailer::Base.deliveries = []
 
     @app = create(:application)
-    @bi = create(:batch_invitation, supported_permission_ids: [@app.signin_permission.id])
+    @bi = create(:batch_invitation)
```

Comment:
Does this work as `@bi = create(:batch_invitation, supported_permissions: [@app.signin_permission])`?